pkg/workflows/upgrade.go

```diff
@@ -238,6 +238,11 @@ func (s *upgradeCoreComponents) Run(ctx context.Context, commandContext *task.Co
 	}
 	commandContext.UpgradeChangeDiff.Append(changeDiff)
 
+	if err = commandContext.AddonManager.Install(ctx, commandContext.ManagementCluster, commandContext.CurrentClusterSpec, commandContext.ClusterSpec); err != nil {
```

Comment:
nit: consistency with above `if err != nil` error handling as opposed to `if err =`

Reply:
reason not updating other methods in this step is bc all other methods return `changeDiff, err` instead of just err